packages/react-core/src/components/ProgressStepper/__tests__/ProgressStep.test.tsx

```diff
@@ -138,7 +136,7 @@ test('Renders with aria-labelledBy on Component element when id and titleId are
       Test
     </ProgressStep>
   );
-  expect(screen.getByText('Test')).toHaveAttribute('aria-labelledby', 'test-id title-id');
+  expect(screen.getByText('Test')).toHaveAccessibleName('Test');
 });
```

Comment:
Small nit: could you rework these test names to reflect what's really being tested here now? We aren't really checking for `aria-labelledBy` anymore, just the accessible name behavior.